### PR TITLE
[shopsys] add upgrade notes for persoo article feed

### PR DESCRIPTION
**Pull request by vitek-rostislav**
| Q             | A
| ------------- | ---
|Description, reason for the PR| added missing upgrade notes for https://github.com/shopsys/shopsys/pull/2940
|New feature| Yes 
|[BC breaks](https://docs.shopsys.com/en/latest/contributing/backward-compatibility-promise/)| No 
|Fixes issues| ... 
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

----
:globe_with_meridians: Live Preview:
  - https://rv-persoo-article-upgrade-notes.odin.shopsys.cloud
  - https://cz.rv-persoo-article-upgrade-notes.odin.shopsys.cloud

### CODE REVIEW

File: UPGRADE-14.0.md [hunk 1]
```diff
@@ -629,6 +629,131 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
...
+        ) {
+        }
+    ```
+    -   `Shopsys\FrameworkBundle\Component\Image\ImageFacade::__construct()` changed its interface:
+    ```diff
+        public function __construct(
+            // ...
+            protected readonly LoggerInterface $logger,
+            protected readonly CdnFacade $cdnFacade,
+    +       protected readonly ImageRepository $imageRepository,
```
Comment (grossmannmartin) [inline UPGRADE-14.0.md]: In the source code I see a different change here – 
Snímek obrazovky 2024-01-04 v 8 31 30

File: UPGRADE-14.0.md [hunk 2]
```diff
@@ -629,6 +629,131 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
...
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::getById()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::getAllArticlesCountByDomainId()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::getOrderedArticlesByDomainIdAndPlacementQueryBuilder()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::create()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::edit()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::delete()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::saveOrdering()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::getAllByDomainId()` is now strictly typed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::getVisibleById()` was removed
+    -   `Shopsys\FrameworkBundle\Model\Article\ArticleFacade::getVisibleArticlesForPlacementOnCurrentDomain()` was removed
```
Comment (grossmannmartin) [inline UPGRADE-14.0.md]: in the rest of the file, it's more common to group changes by class
e.g.

```
- `Shopsys\FrameworkBundle\Model\Article\ArticleFacade`
    - method `findById()` is now strictly typed
    - method `getById()` is now strictly typed
    ...
```

I find it clearer, what do you think?

File: UPGRADE-14.0.md [hunk 3]
```diff
@@ -629,6 +629,131 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
...
+        public function __construct(
+            protected readonly ImageFacade $imageFacade,
+            protected readonly ImageConfig $imageConfig,
+            protected readonly Domain $domain,
+    -       protected readonly FrontendApiImageFacade $frontendApiImageFacade,
+    +       protected readonly ImageApiFacade $imageApiFacade,
+    +       protected readonly DataLoaderInterface $imagesBatchLoader,
+    +       protected readonly DataLoaderInterface $firstImageBatchLoader,
+    ```
+    -   `Shopsys\FrontendApiBundle\Model\Resolver\Image\ImagesQuery::imagesByEntityQuery()` has been removed, use `imagesByEntityPromiseQuery()` instead
```
Comment (grossmannmartin) [inline UPGRADE-14.0.md]: FrontendApi Image Facade was removed, should it be mentioned?

**Review by grossmannmartin — CHANGES_REQUESTED** [review]

File: UPGRADE-14.0.md (hunk 3 above)
Comment (vitek-rostislav) [reply UPGRADE-14.0.md]: ah, good catch, I missed that :+1:

File: UPGRADE-14.0.md (hunk 2 above)
Comment (vitek-rostislav) [reply UPGRADE-14.0.md]: I do not have any strong opinion on this, so I accept your suggestion :slightly_smiling_face:

File: UPGRADE-14.0.md (hunk 1 above)
Comment (vitek-rostislav) [reply UPGRADE-14.0.md]: oh, good catch, thank you! Fixed

**Review by grossmannmartin — APPROVED** [review]

**Pull request merged**